Petar_Nedyalkov/week2/task1.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <fstream>
+
+int main()
+{
+	std::ofstream file("result.txt");
+
+	if (!file.is_open())
+	{
+		std::cout << "Error!" << std::endl;
+		return -1;
+	}
+
+	int a, b, c;
+	int sum = 0, multi = 1;
+	std::cin >> a >> b >> c;
+	sum = a + b + c;
+	multi = a * b * c;
+
+	file << sum << " " << multi << std::endl;
+	file.close();
+	
+	std::ifstream fileO("result.txt");
+
+	if(!fileO.is_open())
```

Comment:
You can also explore the idea of creating a simple function called `assertOpen` that throws exception so that you don't need to write this boring `if` every time :P